site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,522 @@
+    </Tabs.TabPanel>
+    <Tabs.TabPanel>
+      A high-level educational institution in which students study for degrees and academic research is done.
+    </Tabs.TabPanel>
+  </Tabs>
+</PlaygroundPreview>
+
+<ul style="margin-bottom:0;">
+  <li>
+    The learn more about the <strong>Tabs</strong> component, please refer to
+    <InternalLink href="/components/tabs">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 6. Main content
+
+Following the hero section, the main content area is where the primary information and functionality of the page are presented. This section should be well-structured, with headings, subheadings, and other design elements that guide users through the content.
+
+<ul style="margin-bottom:0;">
+  <li>
+    The learn more about the <strong>heading hierarchy</strong>, please refer to
+    <InternalLink href="/foundation/design-tokens/typography#heading-sizes">the Typography page.</InternalLink>
+  </li>
+  <li>
+    The learn more about the <strong>SideNavigation</strong> component, please refer to
+    <InternalLink href="/components/side-navigation">the component page.</InternalLink>
+  </li>
+  <li>
+    The learn more about the <strong>Tabs</strong> component, please refer to
+    <InternalLink href="/components/tabs">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 7. Footer
+
+The footer is placed at the bottom of the site and includes supplementary information and links, such as contact details, legal information, social media icons, and secondary navigation elements. The footer helps users find additional resources and reinforces the overall site structure by providing a consistent and accessible navigation area on every page.
+
+<PlaygroundPreview>
+  <Footer title="Helsinki Design System" footerAriaLabel="HDS Footer">
+    <Footer.Navigation navigationAriaLabel="HDS Footer navigation">
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+      <Footer.Item label="Item" />
+    </Footer.Navigation>
+    <Footer.Utilities backToTopLabel="Back to top">
+      <Footer.SoMe>
+        <Footer.Item icon={<IconFacebook />} />
+        <Footer.Item icon={<IconTwitter />} />
+        <Footer.Item icon={<IconInstagram />} />
+        <Footer.Item icon={<IconYoutube />} />
+        <Footer.Item icon={<IconTiktok />} />
+      </Footer.SoMe>
+      <Footer.Item label="Contact us" />
+      <Footer.Item label="Give feedback" />
+    </Footer.Utilities>
+    <Footer.Base copyrightHolder="Copyright" copyrightText="All rights reserved">
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+      <Footer.Item label="Link" />
+    </Footer.Base>
+  </Footer>
+</PlaygroundPreview>
+
```

Comment:
Should we have some kind of minimum (Header and footer) and maximum of navigation elements examples?

Reply:
Yeah, maybe a sort of default set of elements and then max and min sets. It would serve as an example how our DS scales to different needs.